notebook/gateway/managers.py

```diff
@@ -503,20 +575,23 @@ def shutdown_all(self, now=False):
             self.remove_kernel(kernel_id)
 
 
-
 class GatewayKernelSpecManager(KernelSpecManager):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        base_endpoint = url_path_join(GatewayClient.instance().url,
-                                      GatewayClient.instance().kernelspecs_endpoint)
-
-        self.base_endpoint = GatewayKernelSpecManager._get_endpoint_for_user_filter(base_endpoint)
-        self.base_resource_endpoint = url_path_join(GatewayClient.instance().url,
-                                                    GatewayClient.instance().kernelspecs_resource_endpoint)
+        some = {
```

Comment:
Is ```some``` being used anywhere ? Also please change the name to something specific.

Reply:
Nope removed it.